File: SDKs/googleapis.com/cloudscheduler/v1beta1/java/lib/src/main/java/openapisdk/models/shared/RunJobRequest.java
package openapisdk.models.shared;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;

/**
 * RunJobRequest
 * Request message for forcing a job to run now using RunJob.
**/
public class RunJobRequest {
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("legacyAppEngineCron")
    public Boolean legacyAppEngineCron;
    public RunJobRequest withLegacyAppEngineCron(Boolean legacyAppEngineCron) {
        this.legacyAppEngineCron = legacyAppEngineCron;
        return this;
    }
}

File: SDKs/googleapis.com/cloudscheduler/v1beta1/python/src/sdk/models/shared/runjobrequest.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from sdk import utils


@dataclass_json
@dataclasses.dataclass
class RunJobRequest:
    r"""RunJobRequest
    Request message for forcing a job to run now using RunJob.
    """
    
    legacy_app_engine_cron: Optional[bool] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('legacyAppEngineCron') }})
    
